src/adapters/scene.js

```diff
@@ -120,7 +136,7 @@ Scene.prototype.fitMap = function(poi, options = {}) {
       this.mb.fitBounds(poi.bbox, {padding : padding, animate : false})
     }
   } else {
-    let flyOptions = {center : poi.getLngLat(), screenSpeed: 1.5, animate:false}
+    let flyOptions = {center : poi.getLngLat(), screenSpeed: 1.5, anigithubmate:false}
```

Comment:
you have misspelled "i write on the bad window" :)

Reply:
oops